pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
+, enableZlib ? false, zlib
+, enablePng ? false, libpng
+, enableEigen ? false, eigen }:
+
+stdenv.mkDerivation rec {
+  version = "1.1";
+  name = "openmvg-${version}";
+
+  src = fetchgit {
+    url = "https://www.github.com/openmvg/openmvg.git";
+
+    # Tag v1.1
+    rev = "f5ecb48";
+    sha256 = "1di9i7yxnkdvl8lhflynmqw62gaxwv00r1sd7nzzs9qn63g0af0f";
+    fetchSubmodules = true;
+  };
+
+  buildInputs =
+       [ glibc ]
```

Comment:
Not sure. Haven't tried it, although I generally prefer to be as atomic as possible and assume as little as possible about the current build system at hand. Will the compile break if `glibc` (or another appropriate POSIX-provider) is not passed in? As in, will NixOS inherently include this if `stdenv` is already included?